### ISSUE TITLE
Standardize the appearance of the height profile window

### ISSUE THREAD

**Issue by valleyofdawn** [IsraelHikingMap/Site#346]
"סטטיסטיקת מסלול" is stating the obvious and is taking up useful map space by unnecessarily extending the top of the window.

**Comment by valleyofdawn** [comment]
I just saw you said it's needed for moving the window. In that case just decrease the font size. Can you make the window a bit more like a "windows"  or "Mac OS" windows, which most users are used to? That is x on the top right, no double lines on the sides, narrower top strip?

**Comment by HarelM** [comment]
I want to keep the opacity, What I can add is a white line beneath the header, but I don't think it'll help much:
I can probably move the 'x' a bit so it will look better...
![image](https://cloud.githubusercontent.com/assets/3269297/25103376/fcdfa4d0-23c4-11e7-925c-5cc19672b131.png)

**Comment by valleyofdawn** [comment]
Why can't the x be on the top right as expected?
Are the "=" symbols on the sides necessary? 
Maybe the KM POIs need to be in the route properties dialogues.

**Comment by valleyofdawn** [comment]
![height graph](https://cloud.githubusercontent.com/assets/689201/25178957/8f2ad386-2510-11e7-8815-cbb44e04c426.png)

**Comment by zstadler** [comment]
The screenshots are showing two different lengths and info: near the graph for a specific point on it and on the bottom of the regular window for the entire track. 

We need do something in order to clarify that the numbers shown near the graph are referring to the vertical "cursor" line. Some options are:
change the title of the graph to "track analysis", use same colur for cursor line and its info, move the info together with to the cursor.

**Comment by valleyofdawn** [comment]
I think that dynamically it's pretty clear as the data changes when you move the vertical line and since height and slope are irrelevant to the whole course.

**Comment by HarelM** [comment]
> Why can't the x be on the top right as expected?

It is consistent with the rest of the x in the app.
>Are the "=" symbols on the sides necessary?

Yes to allow the user to change the window size, this is also useful when using mobile devices.
>Maybe the KM POIs need to be in the route properties dialogues.

Maybe, both places aren't very good IMO, but to my surprise, I learned that people use it, I don't have a better idea where to put it.
>The y axis title is too far from the axis.

Was unable to change this behavior, it's css and I don't like playing with it.

About the screen shots:
Adding the white bar misses the point of transparency IMO, which is one of the things this graph is for.
>We need do something in order to clarify that the numbers shown near the graph are referring to the vertical "cursor" line.

When using the desktop the mouse is also there, I think it's relatively intuitive that the place of the mouse with the horizontal white line and the numbers shown are related to each other.
When using the mobile the experience is not so good, but you can click the chart and see the relevant data.

To summarize, I know the graph is not very eye-pleasing but I don't feel the arguments above will make it better considering the requirements - to be subtle.

**Comment by valleyofdawn** [comment]
Maybe the transparency doesn't serve much purpose, in that case, we should make it consistent with the side window.

**Comment by HarelM** [comment]
@valleyofdawn I'm fairly satisfied with the current look of the height profile window.
I think the transparency is nice and keep the map uncrowded.
I don't think the white border will help much.
I know it's a matter of opinion so I'm not sure how to continue with this issue...

**Comment by HarelM** [comment]
Close in favor of #444.
